graphrag_sdk/tests/test_chunking_strategies.py

```
@@ -0,0 +1,296 @@
+        assert len(result.chunks) >= 1
+        assert "context_prefix" in result.chunks[0].metadata
+        assert "original_chunk" in result.chunks[0].metadata
+
+
+# ---------------------------------------------------------------------------
+# 4–7. Llama strategies (skipped when [llama] extra not installed)
+# ---------------------------------------------------------------------------
+
+_llama_core = pytest.importorskip("llama_index.core", reason="graphrag-sdk[llama] not installed")
+
+import os as _os
```

Ruff (0.15.4)

[error] Module level import not at top of file

(E402)